src/irrlicht_changes/CGUITTFont.cpp

```diff
@@ -1251,21 +1251,31 @@ std::u32string CGUITTFont::convertWCharToU32String(const wchar_t* const charArra
 {
 	static_assert(sizeof(wchar_t) == 2 || sizeof(wchar_t) == 4, "unexpected wchar size");
 
-	if (sizeof(wchar_t) == 4) // Systems where wchar_t is UTF-32
+	if (sizeof(wchar_t) == 4) // wchar_t is UTF-32
 		return std::u32string(reinterpret_cast<const char32_t*>(charArray));
 
-	// Systems where wchar_t is UTF-16:
-	// First, convert to UTF-8
-	std::u16string utf16String(reinterpret_cast<const char16_t*>(charArray));
-	std::wstring_convert<std::codecvt_utf8_utf16<char16_t>, char16_t> converter1;
```

Comment:
`#include <codecvt>` is no longer necessary. The code still compiles.

Reply:
`<locale>` was also added by #13775.